src/radio-tile/radio-tile.scss

```diff
@@ -59,7 +59,7 @@
   user-select: none;
   // Create stacking context
   position: relative;
-  z-index: 0;
```

Comment:
Was the removal of this z-index supposed to be reverted as well?

Reply:
I'm not exactly sure. @FlyersPh9 ?

Reply:
I tried reverting back to `z-index: 0` and tried removing `z-index: 0` without `isolation: isolate;` and I'm not seeing any changes.  Maybe this `z-index` was never needed.